src/components/SeriesNav/SeriesNav.js

```diff
@@ -111,7 +109,19 @@ function SeriesNav({
                 series.title
               )}
             </InfoBoxTitle>
-            <InfoBoxText>{series.description}</InfoBoxText>
+            <InfoBoxText>
+              <span>
+                {series.description}
+                {titlePath && (
+                  <>
+                    {' '}
+                    <Link href={titlePath} passHref>
+                      <Editorial.A>Zur Serien-Übersicht</Editorial.A>
```

Comment:
Is that wording checked with CR/Produktion? We seem to normally write it `Serienübersicht`:
<img width="792" alt="Screenshot 2021-06-15 at 14 43 34" src="https://user-images.githubusercontent.com/410211/122054596-1b5b3100-cde8-11eb-8073-6065d2b2de31.png">
Source: [«Frauenstimmen – Auftakt»](https://www.republik.ch/2021/02/05/auftakt-frauenstimmrecht)

Reply:
yes, and forgot to put it in translate as well XD